docs/_config.yml

```diff
@@ -76,16 +74,6 @@ collections:
   docs:
     output: true
     permalink: /:collection/:path
-  adapters:
```

Comment:
These collections need to be included in the config.

Reply:
Reason being that `sort` (and maybe other?) operations need a collection defined in order to iterate over the set of files.